js-basic/Closure/1-clos.js

```diff
@@ -0,0 +1,20 @@
+function App() {
+  let num1 = 23;
+  function num() {
+    console.log(num1);
+  }
+  return num;
+}
+App()();
+//////neested function store date after destroy for child function
+
```

Comment:
add comments only with double slash what is the sense of using multiple slashes?

Reply:
removed extra slash